src/QCToolsBundle/Resources/views/Default/index.html.twig

```diff
@@ -39,8 +39,12 @@
             </div>
             <div class="projectDescription center-font-xs">
                 <p><abbr title="Quality Control Tools for Video Preservation"><strong>QCTools</strong></abbr> is a software tool that helps users analyze and understand their digitized video files through use of audiovisual analytics and filtering.<br>
-                Funded by the <a href="https://www.neh.gov/" target="_blank">National Endowment for the Humanities</a> and the <a href="http://www.knightfoundation.org" target="_blank">Knight Foundation</a>; designed and led by Dave Rice and the <a href="https://www.bavc.org/" target="_blank">Bay Area Video Coalition</a>; developed by MediaArea, <a href="https://github.com/utzig">Fabio Utzig</a>, <a href="https://github.com/ElderOrb">Alexander Ivash</a>.<br>
-                MediaArea was involved in the initial development and provides binaries for all platforms.</p>
+                Initially Funded by the <a href="https://www.neh.gov/" target="_blank">National Endowment for the Humanities</a> and the <a href="http://www.knightfoundation.org" target="_blank">Knight Foundation</a>; designed and led by Dave Rice and the <a href="https://www.bavc.org/" target="_blank">Bay Area Video Coalition</a>; developed by MediaArea, <a href="https://github.com/utzig">Fabio Utzig</a>, <a href="https://github.com/ElderOrb">Alexander Ivash</a>.<br>
+                Version 1.2 with support of panel tracks and QCVault is brought to you by <a href="https://www.tib.eu">Leibniz Information Centre for Science and Technology (TIB)</a>.<br>
```

Comment:
You could use here as well the shorter `https://tib.eu`.

Reply:
their official URL is with `www`, the URL without `www` redirects to the one with it so slower for the person who clicks on the URL :p.

Reply:
Below you use the short form…

Reply:
> Below you use the short form…

Incoherent, true. Fixed.